MIGRATION_GUIDE.md

```diff
@@ -8,6 +8,16 @@ This migration guide will walk you through the Android SDK v5.0.0 changes as a r
 Under the user-centered model, the concept of a "player" is being replaced with three new concepts: users, subscriptions, and aliases.
 
 
+#### ⚠️ Migration Advisory for current OneSignal customers
```

Comment:
Let's put this either above the **Overview** or under the **Android v5.0.0 Migration Guide**?
Or add some horizontal bars to divide this section as it [blends in](https://github.com/OneSignal/OneSignal-Android-SDK/blob/09e363f34c8a9ece684b880cff17f78289dd5ca5/MIGRATION_GUIDE.md) with the text of the Overview.